src/ecs/task-definition.ts

```diff
@@ -86,18 +86,23 @@ const containerDefinitionFromConfiguration = (params: Params, taskName: string)
   const { region } = variables
   const taskConfig = config.tasks[taskName]
 
-  return {
+  const containerDefinition = {
     name: taskName,
     image: taskConfig.image,
     command: taskConfig.command,
     portMappings: portMappingsFromConfiguration(taskConfig),
     environment: environmentFromEnvVars(envVars),
     secrets: secretsFromConfiguration(taskName, clusterName, config, region),
     logConfiguration: logConfigurationFromConfiguration(taskName, variables),
-    essential: true,
+    essential: taskConfig.essential || true,
```

Comment:
isn't `false || true` always `true` ? should we use `true` only if `essential` is `undefined || null`?